src/main/java/com/google/sps/servlets/AnswerQuestionServlet.java

```diff
@@ -46,7 +67,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DocumentReference gameInstanceDocRef = firestoreDb.collection("gameInstance").document(gameInstanceId);
 
     boolean isQuestionActive = isQuestionActive(questionId, gameInstanceDocRef);
-
+    
+    //ToDo: Check is currentQuestionActive (if students can answer)
```

Comment:
Why this TODO? It already knows if it's active or not from the previous line right?